CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+We are open to, and grateful for, any contributions made by the community.  By contributing to Redux Toolkit, you agree to abide by the [code of conduct](https://github.com/reduxjs/redux-toolkit/blob/master/CODE_OF_CONDUCT.md).
+
+## Reporting Issues and Asking Questions
+Before opening an issue, please search the [issue tracker](https://github.com/reduxjs/redux-toolkit/issues) to make sure your issue hasn't already been reported.
+
+Please ask any general and implementation specific questions on [Stack Overflow with a Redux tag](http://stackoverflow.com/questions/tagged/redux?sort=votes&pageSize=50) for support.
+
+## Development
+
+Visit the [Issue tracker](https://github.com/reduxjs/redux-toolkit/issues) to find a list of open issues that need attention.
+
+Fork, then clone the repo:
+```
+git clone https://github.com/your-username/redux-toolkit.git
+```
+
+### Building
```

Comment:
I think this section needs to be updated to match the actual build scripts we have.  We don't have separate commands for different module types - we have one `build` command that generates everything.

Reply:
Done. It still create CommonJS and UMD?

Reply:
Yeah, you can see we pass those args to TSDX.